### ISSUE TITLE
Attaching notes to channel participants

### ISSUE THREAD

**Issue by alexolog** [EionRobb/purple-discord#130]
The Discord web interface allows attaching notes to channel participants who are not "friends".
Please support this functionality.

**Comment by bobbybee** [comment]
Use case?

**Comment by EionRobb** [comment]
Don't we show this info in the "Get Info" popup?

**Comment by alexolog** [comment]
@bobbybee, Use case: Keeping notes on people that I converse with on a public channel but not privately.  Human memory is limited.

@EionRobb, Show yes, edit no.

**Comment by EionRobb** [comment]
@alexolog Have you tried the "Buddy Notes" plugin?

**Comment by alexolog** [comment]
@EionRobb, Yes, I have installed and activated.  There are two problems with it:
1. It does not work with non-buddies, which is what the original request is about (note: amended due to point 2 below).

2. It only keeps the notes locally instead of getting/setting them on the Discord server.  I am asking for server-side functionality.

Consider the following:

If I log in to the Discord server from somebody else's computer (using the web interface) and add a note to a friend or a non-friend, I expect to be able to see the note when using Pidgin.  And similarly the other way around.

I edited toe original ticket to include both friends and non-friends.

**Comment by alexolog** [comment]
Can this please be looked at?
